src/test/rpc_tests.cpp

```diff
@@ -614,40 +614,26 @@ BOOST_AUTO_TEST_CASE(rpc_arg_helper)
 
     //! Check that `self.Arg` returns the same value as the `request.params` accessors
     RPCHelpMan::RPCMethodImpl check_positional = [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue {
```

Comment:
In commit "rpc: Remove index-based Arg accessor" (fadb3eb57b4d207a678067b89caa45abf1f93702)

Should s/check_positional/check_named/

Reply:
> I think this was called `check_positional` because ... 

Indeed, that was the rationale.

> so maybe a name like `check_args`

or `check_equivalence`? (no big deal either way ofc)